crates/forge/src/scarb.rs

```diff
@@ -50,16 +72,63 @@ pub struct ForgeConfig {
     pub fuzzer_seed: Option<u64>,
 
     #[serde(default)]
-    pub fork: Vec<ForkTarget>,
+    pub fork: Vec<RawForkTarget>,
 }
 
 #[derive(Deserialize, Debug, PartialEq, Default, Clone)]
-pub struct ForkTarget {
+pub struct RawForkTarget {
     pub name: String,
     pub url: String,
     pub block_id: HashMap<String, String>,
 }
 
+impl TryFrom<RawForgeConfig> for ForgeConfig {
```

Comment:
I wonder if it would be possible to resolve by having some kind of custom deserializer just for the `ForkTarget` fields. Without duplicating all structs etc.

Should be investigated if `serde` provides such a functionality.

Reply:
Was wondering about implementing `Serde` manually too, however I believe I would have to use `Serde` api instead of manually converting the already deserialized output. Can investigate on that

Reply:
https://serde.rs/deserialize-struct.html it is quite complex. It is probably easier to leave it this way